src/Makefile

```diff
@@ -76,17 +75,12 @@ $(CLEANDIRS):
 	$(MAKE) -s -C $(@:clean-%=%) clean
 
 phpvendors:
-	$(COMPOSERTMP)/composer install -q
-
-composer_download:
-	curl -sS https://getcomposer.org/installer | php
-	mkdir -p $(COMPOSERTMP)
-	mv composer.phar $(COMPOSERTMP)/composer
+	composer install -q
 
```

Comment:
Problem is that we're loosing the 2 steps approach here (download target and usage target)

Download target should be invoked during an environment preparation phase, not during build. Usage when we need it during build.

Also using a full path name for calling composer is not necessarily an issue, (instead of using the PATH which can be wrong) as long as we don't use something controversial such as /tmp/something. It could have been stored in the fossology tree as well, but I didn't found a variable describing it in that Makefile, so that's why I didn't use it.